### ISSUE TITLE
Randomly lost connection in v1.2.0

### ISSUE THREAD

**Issue by dazre** [getnamo/socketio-client-unreal#166]
Hi,

Month ago we switch to 4.23 UE version with 1.2.0 version of plug-in (we use github version of engine) and found that persistent connections randomly lost. Plug-in uses in dedication engine server to connect with our backend servers. Simultaneously we starts about 10 dedicated servers and half of them work well meantime another half of servers works but did not receive any messages from backend. 

Before we switched to 4.23 we use 4.20 and 0.9.3 for plugin and our servers held working connections months. 

Another problem is that we have not any errors or crashes so we could not provide any logs or something else.

**Comment by getnamo** [comment]
v1.0.0+ uses a new boost-less third party lib. It's possible during certain conditions that it may lose a connection I suppose, what kind of data sizes/frequency are we talking about? There must be a way to reliably trigger this disconnect

**Comment by dazre** [comment]
We are talking about 2.5 KBps in 20-30 bidirectional messages. But we found that connection lost in idle time. 

I built 0.9.3 plug-in version for 4.23 engine and got the same issue. Then a game start using socket connection immediately after start it works fine. But if the game didn't use connection immediately after start - game is waiting for command from upstream - after unknown amount of time connection become unusable. Neither UE4.23 server no upstream server didn't get any errors or disconnect events but no one message sent from one didn't receive by another.
